VSRAD.Package/Options/DebuggerOptions.cs

```diff
@@ -38,6 +38,9 @@ public sealed class DebuggerOptions : DefaultNotifyPropertyChanged
         private bool _singleActiveBreakpoint = false;
         public bool SingleActiveBreakpoint { get => _singleActiveBreakpoint; set => SetField(ref _singleActiveBreakpoint, value); }
 
+        private uint _groupSize = 512;
+        public uint GroupSize { get => _groupSize; set => SetField(ref _groupSize, value == 0 ? 512 : value); }
```

Comment:
The assigned default value of `private` member is meaningless because it is verified in SetFeld.
```suggestion
        private uint _groupSize;
        public uint GroupSize { get => _groupSize; set => SetField(ref _groupSize, value == 0 ? 512 /* default value */ : value); }
```